server/src/main/java/org/candlepin/controller/CandlepinPoolManager.java

```diff
@@ -326,8 +325,8 @@ private void addProductContentToMap(Map<String, Content> contentMap, Product pro
 
             if (existing != null && !content.equals(existing)) {
                 log.warn(
-                    "Multiple versions of the same content found on a single subscription: {}.{}",
-                    (content.getOwner() != null ? content.getOwner().getId() : null), content.getId()
+                    "Multiple versions of the same content found on a single subscription: {}",
+                    content.getId()
```

Comment:
I know this is old, but I've often wondered what the effect of this is on the overall process. Might be worth adding a little more context to the warning. Thoughts?